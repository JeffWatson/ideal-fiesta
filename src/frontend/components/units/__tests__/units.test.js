import React from 'react';
import { shallow } from 'enzyme';

import Artillery from '../artillery';
import SmallTank from '../smallTank';
import Soldier from '../soldier';

const props = {
  player: 'BLUE',
};

describe('frontend/components/units ALL UNIT tests', () => {
  it('should render Artillery correctly', () => {
    const unit = shallow(<Artillery {...props} />);

    expect(unit).toMatchSnapshot();
  });

  it('should render SmallTank correctly', () => {
    const unit = shallow(<SmallTank {...props} />);

    expect(unit).toMatchSnapshot();
  });

  it('should render Soldier correctly', () => {
    const unit = shallow(<Soldier {...props} />);

    expect(unit).toMatchSnapshot();
  });
});
