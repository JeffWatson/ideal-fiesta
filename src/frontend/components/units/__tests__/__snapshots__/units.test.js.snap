// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frontend/components/units ALL UNIT tests should render Artillery correctly 1`] = `
<div
  className="unit artillery BLUE"
>
  Artillery!
</div>
`;

exports[`frontend/components/units ALL UNIT tests should render SmallTank correctly 1`] = `
<div
  className="unit smallTank BLUE"
>
  SmallTank!
</div>
`;

exports[`frontend/components/units ALL UNIT tests should render Soldier correctly 1`] = `
<div
  className="unit soldier BLUE"
>
  Soldier!
</div>
`;
